### PR TITLE
assorted repo housekeeping things (cypress, env files, etc)

### PR DESCRIPTION
**Pull request by tjheffner**
## Description
- Updates the env file vars provided in-repo
- Updates cypress to use env vars for testing
- Updates the fallback base endpoint to use local cms instead of a nonexistent endpoint (`cms.va.gov` ??)
- Extracts the env handling to a script for DRY across the different config files
- Fixes one a11y error related to banners and aria roles. The a11y tests are disabled because there are some separate errors related to aria roles (the header has `role="banner"` and so do the shadow DOM for other banners (PACT Act, etc). Not sure why this isn't reported on the existing site.
- Adds some new e2e tests for news story + story listing pages

## Testing done
- `yarn test`
- `yarn export` then `yarn export:serve`. in a different terminal window `yarn test:cypress`

## QA steps
What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?

```[tasklist]
- [ ] Automated tests have passed
- [ ] Tugboat environment was generated without errors
```

### CODE REVIEW

File: cypress/e2e/newsStory.cy.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/// <reference types="cypress" />
+
+context('News Story', () => {
+  it('News Story page renders with navigation back to parent story list', () => {
+    cy.visit('/butler-health-care/stories/its-flu-shot-time/')
+
+    cy.get('#news-stories-listing-link').click()
+    cy.location('pathname').should('equal', '/butler-health-care/stories/')
+  })
```
Comment (ryguyk) [inline cypress/e2e/newsStory.cy.js:9]: Just noting here that we'll want specific tests for Lovell pages, as this link is modified in the bifurcation process.

File: tsconfig.json [hunk 2]
```diff
@@ -38,8 +38,9 @@
     "*.{ts|tsx|js|jsx}",
     "scripts/*",
     "test/*",
-    "cypress/*",
-    ".storybook/*"
+    "cypress/**/*",
+    ".storybook/*",
+    "*.ts"
```
Comment (ryguyk) [inline tsconfig.json:43]: Do we need this `"*.ts"`? Isn't it captured in line 38? (`"*.{ts|tsx|js|jsx}"`)

**Review by ryguyk — APPROVED** [review]
Approved with limited context/understanding around some of the pieces. I did add one note and one question.

File: tsconfig.json (hunk 2 above)
Comment (tjheffner) [reply tsconfig.json:43]: no we don't & you're right it is, leftover weirdness. will remove

File: cypress/e2e/newsStory.cy.js (hunk 1 above)
Comment (tjheffner) [reply cypress/e2e/newsStory.cy.js:9]: yes definitely. I'll stub some of those out soon

**Review by jtmst — APPROVED** [review]

**Pull request merged**